docs/1000.yaml

```
@@ -141,8 +151,6 @@ Q207244:
 Q208416:
   title: Independence of the continuum hypothesis
   url: https://github.com/flypitch/flypitch
```

```suggestion
  url: https://github.com/flypitch/flypitch/blob/d72904c17fbb874f01ffe168667ba12663a7b853/src/summary.lean#L90
```
maybe let's keep this information in the URL?

Otherwise LGTM

bors d+

---

Good idea!